Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:lts-alpine3.17
+
+WORKDIR /usr/src/app
+
+COPY package.json .
+
+RUN yarn install
+
+COPY . .
+
+EXPOSE 6000
+
+CMD ["yarn", "start"]
```

Comment:
Don't add this line to the image. Why? Because this image will be used for the worker and this is not how we will start the worker.

The application should be started in the docker-compose file which you already have. In production we will configure the server to run the correct command to start the app.